cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/Transactions.java

```diff
@@ -66,6 +70,52 @@ public static TransactionExecutor createTransactionExecutor(TransactionExecutorF
     return factory.createExecutor(ImmutableList.of(txAware));
   }
 
+  /**
```

Comment:
No sure if this can replace all usages of TransactionExecutor. We need a pass on refactoring TransactionExecutor, TransactionExecutorFactory, TransactionContext and Transactional into one consistent way for performing transactional tasks.